src/pages/resultsView/enrichments/AlterationEnrichmentsContainer.tsx

```diff
@@ -125,8 +125,9 @@ export default class AlterationEnrichmentContainer extends React.Component<IAlte
 
                               const groups = _.map(data.groupsSet);
 
-                              const group1 = groups[0];
-                              const group2 = groups[1];
+                              // we want to order groups according to order in prop.groups
+                              const group1 = groups.find((group)=>group.name===this.props.groups[0].name);
```

Comment:
this should probably be improved.  groupSet itself (data passed to column renderer) should have ordering and color info, but it doesn't and need to get this done

Reply:
it's NBD since you're doing it a small constant number of times (2)